addons/mail/static/src/model/model_manager.js

```diff
@@ -1258,19 +1234,14 @@ export class ModelManager {
      * this inverse relation.
      *
      * @private
-     * @param {Object} Models
      */
-    _processDeclaredFieldsOnModels(Models) {
+    _processDeclaredFieldsOnModels() {
```

Comment:
Isn't it possible to avoid all this diff by giving this.models as param like before?
Here the less we use this the better I think.

Reply:
I did it to avoid having to pass Models to every function.

Reply:
IMO it's less intimidating having function with no param and rely on `this` in this case.
I have my problems with `this` but here I prefer relying on `this`, especially things in `this` can be seen as fields on model manager.

This is a small detail anyway, so I wouldn't mind if we chose to keep passing param essentially to reduce diff.